README.md

```diff
@@ -279,7 +86,7 @@ poetry run pytest . -k "not slow and not extras"
 
 ## Documentation
 
-The GQLAlchemy documentation is available on [memgraph.com/docs/gqlalchemy](https://memgraph.com/docs/gqlalchemy/).
+The GQLAlchemy documentation is available on [GitHub](https://github.com/memgraph/gqlalchemy).
 
```

Comment:
I would add information on building the docs with mkdocs and testing locally for our devs. I would say that the reference guide is generated from the code while other parts of the docs are written and located at docs directory. To test the documentation locally, do _this this and that_

Reply:
I'll add this to the README.

Reply:
@katarinasupe I've added the instructions.